oic.r.pstat.raml

```diff
@@ -50,15 +51,12 @@ traits:
       Sets or updates device provisioning status data.
     body:
       application/json:
-        schema: Pstat
+        schema: Pstat-Update
```

Comment:
Similar comment here as for /doxm related to a) 201 not being a valid response and b) removing the PUT section (to be verified with Mark).